src/css.js

```diff
@@ -117,43 +117,29 @@ function augmentWidthOrHeight( elem, name, extra, isBorderBox, styles ) {
 
 function getWidthOrHeight( elem, name, extra ) {
 
-	// Start with offset property, which is equivalent to the border-box value
-	var val,
-		valueIsBorderBox = true,
+	// Start with computed style
+	var valueIsBorderBox,
 		styles = getStyles( elem ),
+		val = curCSS( elem, name, styles ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
-	// Support: IE <=11 only
-	// Running getBoundingClientRect on a disconnected node
-	// in IE throws an error.
-	if ( elem.getClientRects().length ) {
-		val = elem.getBoundingClientRect()[ name ];
+	// Fall back to uncomputed css if necessary
+	if ( val < 0 || val == null ) {
```

Comment:
> you even had to change this condition for the new logic to work

I did? It seems to me the logic is the same.

Before:
1. Try `getBoundingClientRect`
2. If negative, zero, null, or undefined, try `getCSS`.
3. If negative, null, or undefined, get inline style.

After:
1. Try `getCSS`
2. If negative, null, or undefined, get inline style.

Maybe I'm misunderstanding something, but it seems clear to me I haven't changed that condition.

That said, you may be right about this condition being dead code previously as well. I can check that and include it in this PR, but I still believe it is a separate ticket.